tests/disaster-recovery/sc_arbiter/test_netsplit.py

```diff
@@ -0,0 +1,450 @@
+import pytest
+import logging
+import time
+import ocpnetsplit
+
+from ocs_ci.ocs.exceptions import CommandFailed, CephHealthException
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.node import get_all_nodes, wait_for_nodes_status
+from ocs_ci.helpers.sanity_helpers import Sanity
+from datetime import datetime, timedelta, timezone
+from ocs_ci.ocs.resources.pvc import get_pvc_objs
+from ocs_ci.ocs.resources.pod import (
+    Pod,
+    wait_for_pods_to_be_in_statuses,
+    get_pods_having_label,
+    get_pod_logs,
+    get_pod_node,
+    get_ceph_tools_pod,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def check_for_read_pause(logreader_pods, start_time, end_time):
+    paused = False
+    for pod in logreader_pods:
+        pause_count = 0
+        time_var = start_time
+        pod_log = get_pod_logs(
+            pod_name=pod.name, namespace=constants.STRETCH_CLUSTER_NAMESPACE
+        )
+        logger.info(f"Current pod: {pod.name}")
+        while time_var <= (end_time + timedelta(minutes=1)):
+            t_time = time_var.strftime("%H:%M")
+            if f" {t_time}" not in pod_log:
+                pause_count += 1
+                logger.info(f"Read pause: {t_time}")
+            else:
+                logger.info(f"Read success: {t_time}")
+            time_var = time_var + timedelta(minutes=1)
+        if pause_count > 5:
+            paused = True
+            break
+    return paused
+
+
+def check_for_write_pause(logwriter_pod, log_files, start_time, end_time):
+    paused = False
+    for file_name in log_files:
+        pause_count = 0
+        file_log = logwriter_pod.exec_sh_cmd_on_pod(command=f"cat {file_name}")
+        time_var = start_time
+        logger.info(f"Current file: {file_name}")
+        while time_var <= (end_time + timedelta(minutes=1)):
+            t_time = time_var.strftime("%H:%M")
+            if f"T{t_time}" not in file_log:
+                pause_count += 1
+                logger.info(f"Write pause: {t_time}")
+            else:
+                logger.info(f"Write success: {t_time}")
+            time_var = time_var + timedelta(minutes=1)
+        if pause_count > 5:
+            paused = True
+            break
+    return paused
+
+
+class TestNetSplit:
+    @pytest.fixture()
+    def init_sanity(self):
+        self.sanity_helpers = Sanity()
+
+    @pytest.mark.parametrize(
+        argnames="zones, duration",
+        argvalues=[
+            pytest.param(constants.NETSPLIT_DATA_1_DATA_2, 15),
+            pytest.param(constants.NETSPLIT_ARBITER_DATA_1, 15),
+            pytest.param(constants.NETSPLIT_ARBITER_DATA_1_AND_ARBITER_DATA_2, 15),
+            pytest.param(constants.NETSPLIT_ARBITER_DATA_1_AND_DATA_1_DATA_2, 15),
+        ],
+        ids=[
+            "Data-1-Data-2",
+            "Arbiter-Data-1",
+            "Arbiter-Data-1-and-Arbiter-Data-2",
+            "Arbiter-Data-1-and-Data-1-Data-2",
+        ],
+    )
+    def test_netsplit_cephfs(
```

Comment:
Can we have both cephfs and rbd covered in the same test? Few of the steps in the test, setup, teardown are redundant. Combining these two tests can greatly reduce the efficiency and run time.